config/settings/qa.yml

```diff
@@ -3,14 +3,9 @@ dor_services:
   url: 'https://dor-services-qa-lb.stanford.edu'
 etd_url: 'https://etd-qa.stanford.edu'
 h2_url: 'https://sul-h2-qa.stanford.edu'
-gis:
-  robots_host: 'kurma-robots-qa-01.stanford.edu'
-# for purl_url: there is no purl for qa;  sul-h2-qa uses stage ...
```

Comment:
do not need to ssh into the GIS server anymore, since test data is accessible via the already configured pre-assembly servers